sympy/algebras/quaternion.py

```diff
@@ -15,6 +17,17 @@
 from mpmath.libmp.libmpf import prec_to_dps
 
 
+def _check_norm(elements, norm):
+    """validate if input norm is consistent"""
+    if norm is not None and norm.is_number:
+        if norm.is_positive is False:
+            raise ValueError("Input norm must be positive.")
+
+        numerical = all(i.is_number and i.is_real is True for i in elements)
+        if (numerical and not is_eq(norm**2, sum(i**2 for i in elements))):
```

Comment:
```suggestion
        if numerical and is_eq(norm**2, sum(i**2 for i in elements)) is False:
```

Reply:
For this one in particular I used `not` just so it fit a single line without annoying the linter, would putting `is False` and splitting be more readable?

Reply:
modified suggestion